traja/models/nn.py

```diff
@@ -69,6 +68,29 @@ def get_timeseries_data_loaders(data_frame, sequence_length, train_fraction, bat
 
     return train_loader, validation_loader
 
+def get_transformed_timeseries_dataloaders(data_frame, sequence_length, train_fraction, batch_size, shift):
```

Comment:
We don't need a function that doesn't normalise, since the user will (almost always) want to do so.

Reply:
yes as @WolfByttner pointed out, the data has to be scaled between 0 and 1 while training RNNs. `get_timeseries_dataloaders` method is now replaced by the `get_transformed_timeseries_dataloaders`. Also, I removed the shift variable from the TimeseriesDataset class which is not required.